adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMAdapterVungleUtils.m

```diff
@@ -27,13 +27,19 @@
 
 const CGSize kVNGBannerShortSize = {300, 50};
 GADAdSize GADMAdapterVungleAdSizeForAdSize(GADAdSize adSize) {
-  if ((adSize.size.height >= GADAdSizeMediumRectangle.size.height &&
-      adSize.size.width >= GADAdSizeMediumRectangle.size.width) || 
-      (adSize.size.height == 0 && adSize.size.width == 0)) {
+//  if ((adSize.size.height >= GADAdSizeMediumRectangle.size.height &&
+//      adSize.size.width >= GADAdSizeMediumRectangle.size.width) || 
+//      (adSize.size.height == 0 && adSize.size.width == 0)) {
```

Comment:
Because above lines are commented out, might we need to update below comment:
```
// VungleSDK will return MREC ad for 300 X 250 and greater OR
// for 0,0 provided sizes
```
to 
```
// VungleSDK will return MREC ad for 300 X 250 and greater 
```
Thanks a lot. :)